recipe/meta.yaml

```
@@ -9,12 +9,14 @@ source:
   sha256: 77a540690e24b0305878c37ffd421785a6f7e53c8b5720d211b211de8d0e95da
 
 build:
-  number: 0
+  number: 1
+  noarch: python
 
 outputs:
   - name: importlib-metadata
-    script: build_base.bat  # [win]
-    script: build_base.sh  # [not win]
+    script: build_base.sh
+    build:
+      noarch: python
     requirements:
       host:
         - python
```

```suggestion
        - python >=3
```

This will ensure this is not installable in Python 2 due to the removal of the conditional dependencies.

---

But is this the right call? Is conda-forge dropping Python 2? Because `importlib_metadata` officially still supports it.

---

conda-forge already dropped python 2

---

I can add those constrains, but I don't think it will change much as we already dropped python 2
but if you prefer I can add it, no worries

---

Hmm conda-forge is, but I suspect users still on Python 2.7 and using conda-forge will then install this incorrectly... I think we should keep the constraints around; Python 2.7 users I suspect is already hard as it is, hehehee.